Contracts/v1.0/seller/update-ad/request.json5

```diff
@@ -0,0 +1,51 @@
+//PUT /api/v1.0/{applicationname}/product/update
+
+//Header contains sessionId : FAE04EC0-301F-11D3-BF4B-00C04F79EFBC and userId : 1111
+//Request object
+{
+	//[Mandatory] id of the product
+	"Id":"1c8ac39a-48d0-42fe-b2a1-7eaaa4f2f464",
+	//[Mandatory] Seller id of the product
+	"SellerId":"564215",
```

Comment:
Why everything is mandatory in this request?
What if i just want to update title, do i need to pass everything in this request?
Also product id should be part of url of put type of api you don’t need to pass it in a request..

Reply:
done